src/main/java/nextstep/courses/domain/cover/CoverImage.java

```diff
@@ -0,0 +1,72 @@
+package nextstep.courses.domain.cover;
+
+public class CoverImage {
+    private final Long id;
+    private final int imageSize;
+    private final ImageExtension extension;
+    private final int width;
+    private final int height;
+
+    private static final int MAX_IMAGE_SIZE_MB = 1;
```

Comment:
관례적으로 용량을 표현할 때 바이트 단위로 표현합니다.
```suggestion
    private static final int MAX_IMAGE_SIZE_MB = 1 * 1024 * 1024;
```

Reply:
감사합니다 반영하겠습니다.